src/dbnode/generated/thrift/rpc.thrift

```diff
@@ -55,6 +55,7 @@ service Node {
 	FetchBatchRawResult fetchBatchRaw(1: FetchBatchRawRequest req) throws (1: Error err)
 	FetchBatchRawResult fetchBatchRawV2(1: FetchBatchRawV2Request req) throws (1: Error err)
 	FetchBlocksRawResult fetchBlocksRaw(1: FetchBlocksRawRequest req) throws (1: Error err)
+	IndexHashResult indexHash(1: FetchTaggedRequest req) throws (1: Error err)
```

Comment:
Were we hoping to make these streaming? I thought that would likely be the biggest win avoiding allocating thrift structs, etc especially for time series data that mismatches which would need to be pooled otherwise.